README.md

```diff
@@ -1,15 +1,286 @@
+2021-02-24T20:42:01.535+0100	INFO	controller-runtime.controller	Starting workers	{"controller": "bucket", "worker count": 1}
+2021-02-24T20:42:01.535+0100	DEBUG	ackrt	starting reconciliation	{"kind": "Bucket", "namespace": "default", "name": "test-ack-s3-bucket", "generation": 22, "account": "111174500800", "role": "", "region": "eu-west-2"}
+2021-02-24T20:42:02.151+0100	INFO	ackrt	deleted resource	{"kind": "Bucket", "namespace": "default", "name": "test-ack-s3-bucket", "generation": 22}
+2021-02-24T20:42:02.184+0100	DEBUG	ackrt	removed resource from management	{"kind": "Bucket", "namespace": "default", "name": "test-ack-s3-bucket", "generation": 22}
+2021-02-24T20:42:02.184+0100	DEBUG	controller-runtime.controller	Successfully Reconciled	{"controller": "bucket", "request": "default/test-ack-s3-bucket"}
+2021-02-24T20:42:02.185+0100	DEBUG	ackrt	starting reconciliation	{"kind": "Bucket", "namespace": "testing", "name": "test-ack-s3-bucket-2", "generation": 2, "account": "118455800300", "role": "arn:aws:iam::118455800300:role/s3FullAccess", "region": "eu-west-2"}
+2021-02-24T20:42:02.764+0100	INFO	ackrt	deleted resource	{"kind": "Bucket", "namespace": "testing", "name": "test-ack-s3-bucket-2", "generation": 2}
+2021-02-24T20:42:02.805+0100	DEBUG	ackrt	removed resource from management	{"kind": "Bucket", "namespace": "testing", "name": "test-ack-s3-bucket-2", "generation": 2}
+2021-02-24T20:42:02.805+0100	DEBUG	controller-runtime.controller	Successfully Reconciled	{"controller": "bucket", "request": "testing/test-ack-s3-bucket-2"}
+2021-02-24T20:42:02.805+0100	DEBUG	controller-runtime.controller	Successfully Reconciled	{"controller": "bucket", "request": "default/test-ack-s3-bucket"}
+2021-02-24T20:42:02.805+0100	DEBUG	controller-runtime.controller	Successfully Reconciled	{"controller": "bucket", "request": "testing/test-ack-s3-bucket-2"}
+```
+
+### Testing controllers
+
+
+```bash
+ackdev test unit s3
+```
+
+```bash
+ackdev test unit s3                                                                                                                                                       SIGIN
+go test -v ./...
+?   	github.com/aws-controllers-k8s/s3-controller/apis/v1alpha1	[no test files]
+?   	github.com/aws-controllers-k8s/s3-controller/cmd/controller	[no test files]
+?   	github.com/aws-controllers-k8s/s3-controller/pkg/resource	[no test files]
+?   	github.com/aws-controllers-k8s/s3-controller/pkg/resource/bucket	[no test files]
+?   	github.com/aws-controllers-k8s/s3-controller/pkg/resource/version	[no test files]
+?   	github.com/aws-controllers-k8s/s3-controller/pkg/version	[no test files]
+```
+
+```bash
+ackdev test unit dynamodb ecr s3
+```
+
+```bash
+SERVICE     UNIT-TESTS 
+elasticache ERROR      
+ecr         ERROR      
+s3          PASS       
+dynamodb    PASS
+```
+
+### [Re]Generating controllers
+
+```bash
+ackdev generate controller s3 dynamodb --stream-logs
```

Comment:
same comment as above about not operating on multiple controllers/repos at once. Let's stick to a single service at a time for the `ackdev generate controller` command and stick to Bash for loops for multi-plexing.